review/core/admin.py

```diff
@@ -1,11 +1,11 @@
 from django.contrib import admin
 
 from core.models import Project, ProjectReview, PersonReview, ProjectComment, Settings, ManagerPersonReview, \
-    ManagerProjectComment
+    ManagerProjectComment, Round
 
 
 class ProjectReviewAdmin(admin.ModelAdmin):
-    list_display = ('project', 'reviewee', 'get_reviewers')
+    list_display = ('round', 'project', 'reviewee', 'get_reviewers')
```

Comment:
add round filter

Reply:
https://medium.com/elements/getting-the-most-out-of-django-admin-filters-2aecbb539c9a